numpy/core/src/multiarray/strfuncs.c

```diff
@@ -0,0 +1,107 @@
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
+#define _MULTIARRAYMODULE
+
+#include <Python.h>
+#include <numpy/arrayobject.h>
+#include "strfuncs.h"
+
+
+static PyObject *PyArray_StrFunction = NULL;
+static PyObject *PyArray_ReprFunction = NULL;
+
+/*NUMPY_API
+ * Set the array print function to be a Python function.
+ */
+NPY_NO_EXPORT void
+PyArray_SetStringFunction(PyObject *op, int repr)
```

Comment:
the `_deps` listings should be unnecessary now, nothing actually reads them and for gcc compatible compilers it determines dependencies automatically.